health-monitors/gpu-health-monitor/gpu_health_monitor/tests/test_dcgm_watcher/test_dcgm.py

```
@@ -400,6 +400,74 @@ def test_perform_health_check_connectivity_failure_generic_error(self):
+        incident.error = dcgm_structs.c_dcgmDiagErrorDetail_t()
+        incident.error.msg = "XID 95 detected on GPU 0"
+        incident.error.code = dcgm_errors.DCGM_FR_PCI_REPLAY_RATE
+        incident.entityInfo = dcgm_structs.c_dcgmGroupEntityPair_t()
+        incident.entityInfo.entityGroupId = 0
+        incident.entityInfo.entityId = 0
+        mock_response.incidents[0] = incident
+        dcgm_group_mock.health.Check.return_value = mock_response()
+
+        response, connectivity_success = watcher._perform_health_check(dcgm_group_mock)
+
+        assert connectivity_success == True
```

Ruff (0.15.2)

[error] Avoid equality comparisons to `True`; use `connectivity_success:` for truth checks

Replace with `connectivity_success`

(E712)